src/current/releases/molt.md

```diff
@@ -0,0 +1,155 @@
+- [Live Migration Service]({% link molt/live-migration-service.md %})
+
+Cockroach Labs recommends using the latest available version of each tool. See [Installation](#installation).
+
+<div class="filters filters-big clearfix">
+    <button class="filter-button" data-scope="fetch_verify">Fetch/Verify</button>
+    <button class="filter-button" data-scope="lms">LMS</button>
+</div>
+
+## Installation
+
+<section class="filter-content" markdown="1" data-scope="fetch_verify">
+To download the latest MOLT Fetch/Verify binary:
+
+{% include molt/molt-install.md %}
+
+{{site.data.alerts.callout_info}}
+[MOLT Fetch]({% link molt/molt-fetch.md %}) is supported on Red Hat Enterprise Linux (RHEL) 9 and above.
+{{site.data.alerts.end}}
+</section>
+
+<section class="filter-content" markdown="1" data-scope="lms">
+To install the LMS, refer to [MOLT Live Migration Service]({% link molt/live-migration-service.md %}#installation).
+
+To update the LMS once installed, configure the following [Helm chart values](https://github.com/cockroachdb/molt-helm-charts/blob/main/lms/values.yaml):
+
+{% include molt/update-lms-version.md %}
+
+For more information, refer to [Configuration]({% link molt/live-migration-service.md %}#configuration).
+</section>
+
+<section class="filter-content" markdown="1" data-scope="fetch_verify">
+## July 10, 2024
+
+MOLT Fetch/Verify 1.1.1 is [available](#installation).
+
+- Fixed a bug that led to incorrect list continuation file behavior if a trailing slash was provided in `--bucket-path`.
+- Fixed a bug with extracting the filename from a failed import URL. Previously, an older filename was being used, which could result in duplicated data. Now, the filename that is used in import matches what is stored in the exceptions log table.
+- Added a `--use-implicit-auth` flag that determines whether [implicit authentication]({% link {{ site.current_cloud_version }}/cloud-storage-authentication.md %}) is used for cloud storage import URIs.
+
+## July 8, 2024
+
+MOLT Fetch/Verify 1.1.0 is [available](#installation).
+
+- Fixed a bug where integer values for `JSONB`/`JSON` types were not correctly handled when migrating from a PostgreSQL or CockroachDB source.
+- Schemas can now be created from Oracle sources.
+- Optimized the logic for clearing continuation tokens. Now, the clearing option is only presented in the relevant modes and when there are tokens to clear.
+- CockroachDB sources now exclusively accept connections from `admin`/`root` users.
+- The statement timeout can now be adjusted during the export phase. This guards against hanging queries.
+
+## May 31, 2024
+
+MOLT Fetch/Verify 1.0.0 is [available](#installation).
+
+- Renamed the `--table-splits` flag to `--concurrency-per-table`, which is more descriptive.
+- Increased the default value of `--import-batch-size` to `1000`. This leads to better performance on the target post-migration, but can cause the import to take longer since more data is now imported in each batch.
```

Comment:
The second sentence here may be misleading.  Each individual IMPORT job will take longer, but the sum total of all jobs should take the same (or less) time.

Reply:
Amended the note.